docs/changelog.md

```diff
@@ -83,6 +83,39 @@ For more information on features and bug fixes in 1.0.0, see the GitHub mileston
 
 #### Upgrade Instructions
 
+##### Removal of deprecated classes/methods/functionalities
```

Comment:
As these are all internal classes/methods, do you think it's worth mentioning all of them explicitly in the upgrade instructions?

Reply:
It could be interesting for people that are using internal methods. :see_no_evil: